ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.html

```diff
@@ -1,20 +1,27 @@
-<div class="menu">
-  <app-menu></app-menu>
-</div>
-<yandex-map></yandex-map>
-<mat-button-toggle-group
-	class="btn-group"
-	name="fontStyle"
-	aria-label="Font Style"
->
-	<mat-button-toggle class="passanger-btn" value="bold"
+<yandex-map
+	[tripData]="tripFormData"
+	[userState]="userState"
+	[isSavedRoute]="isSavedRoute"
+	[activeRoutesCollection]="routes"
+></yandex-map>
+<app-trip-registration
+	[isShown]="isHiddenTripRegistration"
+	(formData)="getData($event)"
+	(isShownViewListButton)="getIsShown($event)"
+	(isShownSaveRouteButton)="getIsShownSaveRoute($event)"
+></app-trip-registration>
+<app-routes-list *ngIf="isShownRoutesList" [activeRoutesCollection]="routes"></app-routes-list>
+<mat-button-toggle-group class="btn-group" name="fontStyle" aria-label="Font Style">
+	<mat-button-toggle class="passanger-btn" value="bold" (click)="userState = 'passenger'"
```

Comment:
move to function, it's a bad practice to set string to a variable in the html

Reply:
fixed